bigtable/client/cell.h

```diff
@@ -0,0 +1,81 @@
+// Copyright 2017 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef BIGTABLE_CLIENT_CELL_H_
+#define BIGTABLE_CLIENT_CELL_H_
+
+#include <absl/base/thread_annotations.h>
+#include <absl/strings/string_view.h>
+#include <absl/synchronization/mutex.h>
+#include <bigtable/client/version.h>
+#include <vector>
+
+namespace bigtable {
+inline namespace BIGTABLE_CLIENT_NS {
+/**
+ * The in-memory representation of a Bigtable cell.
+ *
+ * Bigtable stores data in rows, indexes by row keys.  Each row may contain
+ * multiple column families, each column family might contain multiple columns,
+ * and each column has multiple cells indexed by timestamp.  Notice that the
+ * storage is sparse, column families, columns, and timestamps might contain
+ * zero cells.
+ */
+class Cell {
+ public:
+  Cell(absl::string_view row_key, absl::string_view family_name,
+       absl::string_view column_qualifier, int64_t timestamp,
+       // TODO: use something more generic than vector
+       std::vector<absl::string_view> chunks,
```

Comment:
Why use `absl::string_view` here?  Does the caller (whatever is building a Cell) need to retain ownership of the chunks and labels?  If not, consider moving the ownership from the caller to the cell.

Reply:
done